packages/rancher-istio/1.12/rancher-istio/charts/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 annotations:
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: Istio
-  catalog.cattle.io/kube-version: '>= 1.20.0-0 < 1.24.0-0'
```

Comment:
So we should only be able to install this chart in 1.19, 1.20, 1.21, and 1.22?

Reply:
Yes. That's correct. It's according to the istio support matrics: [https://istio.io/latest/docs/releases/supported-releases/](https://istio.io/latest/docs/releases/supported-releases/)